bin/dustc

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 
 var cli = require('cli').enable('glob', 'version'),
+    watchr = require('watchr'),
```

Comment:
I tried swapping them out, but chokidar requires XCode to be installed if you're on Mac because it compiles `fsevents`. This seems kinda icky :(